scripts/globals/gear_sets.lua

```diff
@@ -256,6 +258,27 @@ function HandleHipsterSet(player, gearset, matches)
         -- printf("we have a special snowflake | gearset: %u | mod %u %u", gearset.id, MOD_FASTCAST, modValue);
         player:addGearSetMod(gearset.id, MOD_FASTCAST, modValue);
         return;
+    elseif (gearset.id == 113) then
+        local modValue = 0;
+        --  Phorcys Armor Set: Set Increases STR, VIT, INT, MND. Bonus is +2 Stats for 2 pieces, +5 for 3 pieces, +10 for 4 pieces, and +15 for 5 pieces
+        if (matches > 1 and matches < 3) then
```

Comment:
Isn't this just `if (matches == 2`)? The others are == 3, 4, and 5 respectively.

If you wanted, you could even cut out the if checks altogether with a table since the values are a 1-to-1 correspondence with how many pieces you're wearing.
```
local modValues = {0, 2, 5, 10, 15}
player:addGearSetMod(gearset.id, MOD_STR, modValues[matches]);
```